File: Week_09/58/length_of_last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        res = 0
        for i in range(len(s) - 1, -1, -1):
            if s[i] == ' ':
                if res == 0:
                    continue
                else:
                    break
            else:
                res += 1
        return res

File: Week_09/58/length_of_last_word.go
package main

func lengthOfLastWord(s string) int {
	res := 0
	for i := len(s) - 1; i >= 0; i-- {
		if s[i] == ' ' {
			if res == 0 {
				continue
			} else {
				break
			}
		} else {
			res++
		}
	}
	return res
}
